### PR TITLE
Added Zarr use cases, Tweet mentions, Zarr visualizations and Code snippets.

### PR DESCRIPTION
**Pull request by Mubarraqqq**

### CODE REVIEW

**Comment by Mubarraqqq** [comment]
**Added Zarr use cases, Tweet mentions, Zarr visualization, and Code snippets.** #15

File: _data/Mubarraqqq/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- https://twitter.com/notjustmoore/status/1579536653852766208
+- https://twitter.com/azavea/status/1573036727169871872
+- https://twitter.com/notjustmoore/status/1256232842755014656
+
+
+## Zarr Data Visualisation
+
+#### 30GB [microscopic image](https://twitter.com/notjustmoore/status/1256232842755014656) of covid19 newly published and converted to zarr stored in Embassy S3 and viewed with an [openmicroscopy plugin](https://www.openmicroscopy.org/bio-formats/) :
+
+![image1](/_data/Mubarraqqq/images/Microscopy1.jpg)
```
Comment (sanketverma1704) [inline _data/Mubarraqqq/README.md]: The visualisation that you've added here and all of the visualisations down below are not rendering. Can you please correct it? Thanks!

**Comment by sanketverma1704** [comment]
Hi @Mubarraqqq. Thanks for sending the PR.
I've left a comment above, please have a look at it. Thanks!

File: _data/Mubarraqqq/README.md (hunk 1 above)
Comment (Mubarraqqq) [reply _data/Mubarraqqq/README.md]: Alright. I will do that right away!

Comment (Mubarraqqq) [reply _data/Mubarraqqq/README.md]: Hello **Mr. Sanket**, I have made the required changes to render my visualization.

**Comment by Mubarraqqq** [comment]
Hello Mr. Sanket, I have made the required changes to render my
visualization on my PR [beautiful-zarr/_data/Mubarraqqq at main ·
Mubarraqqq/beautiful-zarr (github.com)
<https://github.com/Mubarraqqq/beautiful-zarr/tree/main/_data/Mubarraqqq>]

**Comment by sanketverma1704** [comment]
Thanks for updating the PR. Looks good now. Merging this now. Feel free to add more stuff as you go.

**Comment by Mubarraqqq** [comment]
Yes, I definitely will!
From contributions to any raising issues I find. But aside from this, is
there any more I can do to leverage my skills to build the future of the
Zarr community cause I am really passionate about the foreseeable future of
Zarr.

**Pull request merged**